### ISSUE TITLE
SecurityError: Attempt to use history.replaceState() more than 100 times per 30 seconds

### ISSUE THREAD

**Issue by gio00** [livewire/livewire#1496]
### Description

I get this error: 

```
SecurityError: Attempt to use history.replaceState() more than 100 times per 30 seconds
```

It's not always happening and my guess is that I have a lot of little components in my view. 

**Context**
- Livewire version: 2.0.0
- Laravel version: 7.0.0
- Browser:  Safari

**Comment by Zaknfn** [comment]
I had the same problem, I solved by updating to 2.2.5:
v2.1.2 => v2.2.5

**Comment by calebporzio** [comment]
Closing as it's been fixed
